usr.bin/procstat/procstat_auxv.c

```diff
@@ -56,6 +58,19 @@
 
 #include "procstat.h"
 
+static const char *
+fmt_ptr(void *ptr)
+{
+	static char ptrstr[128];
+#ifdef __CHERI_PURE_CAPABILITY__
+	if ((procstat_opts & PS_OPT_VERBOSE) != 0)
+		strfcap(ptrstr, sizeof(ptrstr), "%T%C", (uintcap_t)ptr);
```

Comment:
Why do you need `%T` here?

Reply:
I think we need %T because the raw cap exported from the kernel is untagged, but we know it's actually tagged and don't want a bogus "(invalid)" attribute.

FWIW, I would be fine with just printing the caps verbose always and not adding the new -v flag.

Reply:
`%T` and this sort of thing is in fact the original use case for strfcap. :)

I added -v because it seemed like the easy way to make libxo formatted output remain consistent and I think the output is ugly at 80 columns, but I could drop that bit.

Reply:
Ok.  I checked GDB and `info auxv` there only prints the addresses as well (possibly something I should try to fix)